Dockerfile

```diff
@@ -267,6 +267,16 @@ RUN --security=insecure --mount=type=cache,id=testspace,target=/tmp --mount=type
 FROM scratch AS test
 COPY --from=test-runner /src/coverage.txt /coverage.txt
 
+# The test-race target performs tests with race detector.
+
+FROM golang:1.12 AS test-race
```

Comment:
probably golang version here should be pushed to the Makefile

Reply:
I tried to but I can't make it accept `1.12` or full `golang:1.12` as a build arg, giving up. seems that it works only for the first `FROM` in the `Dockerfile`, but it breaks `FROM $tools` if I move this target about `tools`

Reply:
resolved the issue, also documented things a bit so that next one won't have confusion :)